Documentation/sourcebuild-user-onboarding-guide/packaging.md

```diff
@@ -0,0 +1,26 @@
+## Linux Distribution Packaging Guidelines
+
+ If you want to create a Linux distribution package (rpm, deb) out of the source-build SDK here are some suggestions. See [.NET Core distribution packaging](https://learn.microsoft.com/en-us/dotnet/core/distribution-packaging) for information on suggested packages, sub-packages, name and contents. This is the minimal amount of content you need to package up:
+
+1. Extract the tarball to the distribution appropriate location such as /usr/lib64/dotnet/, /usr/lib/dotnet/, or /usr/lib/x86_64-linux-gnu/dotnet/. 
+
+2. Create a symlink from /usr/bin/dotnet (or equivalent) to the dotnet binary in the SDK that you installed in the previous step.   
+For example: ln -s /usr/lib64/dotnet/dotnet /usr/bin/dotnet 
+Now users can simply run dotnet and it will work. 
+
+3. Create an /etc/dotnet/install_location file and add the path of the SDK directory in there. The file should contain a single line like this: /usr/lib64/dotnet  
+ This file is used by [.NET Core to find the SDK/Runtime location](https://github.com/dotnet/designs/blob/main/accepted/2020/install-locations.md). 
+
+4. Define DOTNET_ROOT and update PATH by saving the following as /etc/profile.d/dotnet.sh (or equivalent).
+
+    Set location for AppHost lookup[ -z "$DOTNET_ROOT"] &&exportDOTNET_ROOT=/usr/lib64/dotnet.  
+    
+    Add dotnet tools directory to PATHDOTNET_TOOLS_PATH="$HOME/.dotnet/tools"case"$PATH"in*"$DOTNET_TOOLS_PATH"*) true;; 
+         *) PATH="$PATH:$DOTNET_TOOLS_PATH";; 
+         esac
```

Comment:
```suggestion
    Add dotnet tools directory to PATH: 
      DOTNET_TOOLS_PATH="$HOME/.dotnet/tools"
      export PATH="$PATH:$DOTNET_TOOLS_PATH"
```

Not sure if this was what you were going for but seems incorrect as is.

Reply:
Done